backend/src/test/java/com/bakdata/conquery/util/extensions/GroupExtension.java

```diff
@@ -0,0 +1,22 @@
+package com.bakdata.conquery.util.extensions;
+
+import com.bakdata.conquery.io.storage.MetaStorage;
+import com.bakdata.conquery.models.auth.entities.Group;
+import lombok.Getter;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+@Getter
+public class GroupExtension implements BeforeAllCallback {
```

Comment:
was ist das?

Reply:
Ich habe das für einige Komponenten erstellt, die in einen Storage müssen, noch dem er geöffnet wurde, damit die Logik nicht in den `@BeforeAll`-Callback eines Tests muss und man es wiederverwenden kann.

Wir könnten diesen Extensions-Mechanismus auch um die Cached TestConqueries bauen um dann nicht mehr die DynamicNodes für ProgrammaticTests zu nutzen, sondern sie als echte Tests nutzen.
Der RestartTest könnte vermutlich auch DropwizardAppExtension nutzen